Code/Tools/ProjectManager/Source/GemCatalog/GemItemDelegate.cpp

```diff
@@ -169,6 +173,82 @@ namespace O3DE::ProjectManager
         return QStyledItemDelegate::editorEvent(event, model, option, modelIndex);
     }
 
+    const GemModel* GetGemModel(const QAbstractItemModel* model)
```

Comment:
Rename to GetSourceGemModel or something like that and the the model parameter to proxyModel? Without seeing the implementation I had not understand what this is doing.